src/Symfony/Component/Translation/Remote/RemoteDecorator.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation\Remote;
+
+use Symfony\Component\Translation\TranslatorBag;
+
+class RemoteDecorator implements RemoteInterface
```

Comment:
Why is it useful?

Reply:
It is in the `read` method, that we merge wanted domains (passed as arguments in commands) with default domains (defined in configuration)